src/components/TeamMemberList.js

```diff
@@ -0,0 +1,33 @@
+import { useState, useEffect } from "react";
+import TeamMember from "./TeamMember";
+import axios from "axios";
+
+const TeamMemberList = (props) => {
+  const [users, setUsers] = useState([]);
+
+  useEffect(() => {
+    const fetchUsers = async () => {
+      const response = await axios.get(process.env.REACT_APP_ENDPOINT_URL);
+      setUsers(response.data);
+    };
+    fetchUsers();
+  },[]);
+
+  return (
+    <>
+      {users.map((user) => {
```

Comment:
same thing with loading, this will probably be very fast, but what if someone have wooden pc and this fetch will last 30sec? 
good practice if you fetch on start would be render like this:
loader (start fetching)
array (if success)
error (if error)
good article:
https://www.robinwieruch.de/react-hooks-fetch-data

Reply:
Added loading message. Good article - covered everything I was not sure of :)